src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1755,6 +1755,11 @@ bool ApplicationManagerImpl::Stop() {
   stopping_application_mng_lock_.Acquire();
   is_stopping_ = true;
   stopping_application_mng_lock_.Release();
+
+  if (app_launch_ctrl_.get()) {
```

Comment:
@ZhdanovP Why do you need this `get()`. Use `explicit operator bool() const noexcept;`

Reply:
@AByzhynar Please, take a look ca5542a68eb62bb8ecd0167987c6a58819dca504